lib/balance-tx/test/spec/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -513,8 +511,11 @@ spec_balanceTransaction = describe "balanceTransaction" $ do
     it "does not balance transactions if no inputs can be created"
         $ property prop_balanceTransactionUnableToCreateInput
 
-    it "produces valid transactions or fails"
-        $ property prop_balanceTransactionValid
+    it "produces valid transactions or fails (Babbage)"
+        $ property (prop_balanceTransactionValid @BabbageEra)
+
+    it "produces valid transactions or fails (Conway)"
+        $ property (prop_balanceTransactionValid @ConwayEra)
```

Comment:
👍🏻

Reply:
Yes! Btw, I wanted to have `AnyRecentEra` generated in the property instead of calling it twice, but I didn't find a nice solution